google/cloud/bigquery/model.py

```diff
@@ -434,6 +449,52 @@ def __repr__(self):
         )
 
 
+class TransformColumn:
+    """TransformColumn represents a transform column feature.
+
+    See
+    https://cloud.google.com/bigquery/docs/reference/rest/v2/models#transformcolumn
+
+    Args:
+        resource:
+            A dictionary representing a transform column feature.
+    """
+
+    def __init__(self, resource: Dict[str, Any]):
+        self._properties = resource
+
+    @property
+    def name(self) -> Optional[str]:
+        return self._properties.get("name")
+
+    @property
+    def type_(self) -> Optional[str]:
+        type_json = self._properties.get("type")
+        if type_json is None:
+            return None
+        return standard_sql.StandardSqlDataType.from_api_repr(type_json)
+
+    @property
+    def transform_sql(self) -> Optional[str]:
+        return self._properties.get("transformSql")
+
+    @classmethod
+    def from_api_repr(cls, resource: Dict[str, Any]) -> "TransformColumn":
+        """Constructs a transform column feature given its API representation
+
+        Args:
+            resource:
+                Transform column feature representation from the API
+
+        Returns:
+            Transform column feature parsed from ``resource``.
+        """
+        this = cls(None)
```

Comment:
This method isn't covered by unit tests.

The "cover" session failed.

```
google/cloud/bigquery/model.py                                 197      4     34      0    98%   492-495
```

I'll make a suggested edit that will address this.